public/service-worker.js

```diff
@@ -63,11 +63,17 @@ const getOptions = ({ path, id }) => ({
   tag: id,
   timestamp: Date.now(),
   data: {
-    path, // TODO: at this step change to full url
+    path: getPath({}),
   },
 });
 
-// Add error handling if data will not match
+// TODO: handle beta.near.org version redirection
```

Comment:
@marcinbodnar if we update this TODO, to handle near.org later, and update line 73 to point to beta.near.org,  then we can confirm this works while testing on beta and complete the TODO prior to merging to main

Reply:
I mean, it's not a proper approach, but I guess we can handle it that way. WIll prepare commit.